### PR TITLE
Add ☕️

### PR DESCRIPTION
**Pull request by raganwald**

### CODE REVIEW

**Comment by raganwald** [comment]
Add more :coffee:

**Comment by raganwald** [comment]
Even more.

**Comment by ymendel** [comment]
testing, one two

**Comment by ymendel** [comment]
testing one more time

**Comment by raganwald** [comment]
> testing one more time

Copy, paste, cmd-enter

**Comment by ymendel** [comment]
One more quick test.

**Comment by raganwald** [comment]
And another test, because we :heart: testing!

heart-03

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-safasf
+# Emoji
+
+☕️
```
Comment (raganwald) [inline README.md]: In a comment,w e could just write `:coffee:`.

Comment (raganwald) [inline README.md]: Like this: :coffee:.

Comment (raganwald) [inline README.md]: Or this, which is added in Chrome on the desktop.

chrome

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-safasf
+# Emoji
+
```
Comment (shayfrendt) [inline README.md]: Yeah this whitespace is highly suspect. :trollface:

Comment (raganwald) [inline README.md]: I once tried to joke about the Power of Whitespace. It was a Black day for me.

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
-safasf
```
Comment (raganwald) [inline README.md]: I’m glad we removed this.

File: README.md (hunk 1 above)
Comment (raganwald) [inline README.md]: This has been added.

Comment (raganwald) [inline README.md]: Add one more.

**Pull request merged**